fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -103,3 +115,62 @@ INSTANTIATE_TEST_CASE_P(LogicalTest, Int32LogicalBinary, ::testing::Combine(
         std::make_tuple("iand", iand),
         std::make_tuple("ixor", ixor)
         )));
+
+class Int64LogicalBinary : public TRTest::BinaryOpTest<int64_t> {};
+
+TEST_P(Int64LogicalBinary, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120, "(method return=Int64 ("
+                                "block ("
+                                "lreturn ("
+                                "%s (lconst %li) (lconst %li)"
+                                ") )))", 
```

Comment:
Not quite what I was thinking. :sweat_smile:

Here are some examples of how it's done in other files: 
https://github.com/eclipse/omr/blob/4d1af9870cec7db040cc9a785c8de9c16ae1e814/fvtest/compilertriltest/TypeConversionTest.cpp#L74-L78
https://github.com/eclipse/omr/blob/4d1af9870cec7db040cc9a785c8de9c16ae1e814/fvtest/compilertriltest/CompareTest.cpp#L58-L63